packages/components/src/dropdown/index.js

```diff
@@ -43,8 +43,14 @@ class Dropdown extends Component {
 	 * When contents change height due to user interaction,
 	 * `refresh` can be called to re-render Popover with correct
 	 * attributes which allow scroll, if need be.
+	 * @deprecated
 	 */
 	refresh() {
+		deprecated( 'Dropdown.refresh()', {
+			plugin: 'Gutenberg',
+			version: '4.5',
+			hint: 'Popover is now automatically re-rendered without needing to execute "refresh"',
+		} );
 		if ( this.popoverRef.current ) {
```

Comment:
Does this even need to occur? Could we change the function to a noop? Could be a good opportunity to highlight potential issues, as treating this as a noop forces reliance on the default behavior.

Reply:
what if we just remove the rest of the function, so only `deprecated()` is executed?

Reply:
> what if we just remove the rest of the function, so only `deprecated()` is executed?

Yes, sorry it wasn't clear. I mean precisely that.